Makefile

```diff
@@ -44,7 +48,7 @@ ${VE_DIR}:
 #=> develop: install package in develop mode
 .PHONY: develop
 develop:
-	pip install -e .[dev]
+	pip install -e ".[dev]"
```

Comment:
N.B. zsh treats .[dev] as a glob. Because it doesn't match anything, it returns an error.

Reply:
I'm fairly certain that forgetting the quotes in zsh is considered a [classic blunder](https://www.youtube.com/watch?v=RWW6aDpUvbQ)